src/application/controllers/language/LanguageController.ts

```diff
@@ -0,0 +1,12 @@
+import { FastifyReply, FastifyRequest } from 'fastify'
+import { createLogger } from '@quorum/elisma/src/infra/log'
+import { ProjectLanguage } from '@quorum/elisma/src/domain/bundle/entities/ProjectLanguage'
+
+const logger = createLogger('LanguageController')
+
+export class LanguageController {
+  async getAll(req: FastifyRequest, res: FastifyReply): Promise<void> {
+    logger.info(`Retrieving languages...`)
+    return res.send(Object.values(ProjectLanguage)) //FIXME get it from manifest
```

Comment:
There is an opened PR to add the languages to the manifests. You should use the new ManifestService to retrieve the list of manifests and put all languages in a Set.

Reply:
I'll do it in my next PR